scripts/make-mini-d3.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
+(set -o igncr) 2>/dev/null && set -o igncr; # For Cygwin on Windows compatibility
```

Comment:
can you explain what this does a bit more and maybe put a pointer to the comments? I am asking this because if we hit an issue because of this setting, we then can go back and check the details in the future.